examples/run_twitter_fiction_app.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+
+from api import GPT, Example, UIConfig
+from api import demo_web_app
+
+PROMPT_EXAMPLE_URL = "https://raw.githubusercontent.com/ml4j/gpt-scrolls/master/tweets/twitter-fiction-prompt.json"
```

Comment:
As per comment below,  these external URLs are really for demonstration purposes for this PR review - I'd suggest committing the raw json files as part of this PR should you wish to merge this into the codebase)